charts/dawarich/values.yaml

```diff
@@ -104,6 +105,52 @@ photon:
 #    limits:
 #      memory: 512Mi
 
+  loadbalancerProxy:
+    image:
+      registry: docker.io
+      repository: envoyproxy/envoy
+      tag: v1.31-latest
+      pullPolicy: IfNotPresent
+    podSecurityContext: {}
+    photonHosts:
+      - host: photon.komoot.io
+        scheme: https
+        port: 443
+        rateLimit: 0  # no known limit
```

Comment:
rateLimit: 0 doesn't need to be specified, this can be implied when it is missing. Remove all occurances.